packages/cognito-auth/src/utils/cognito.test.ts

```diff
@@ -90,6 +90,7 @@ describe('Session utils', () => {
     it('should get a session from the cookie if it exists', () => {
       const stringifiedSession = JSON.stringify({
         refreshToken: mockLoginSession.refreshToken,
+        mode: 'DESKTOP',
```

Comment:
Would be nice if we have a test case to cover if `marketplaceGlobalObject` returned by `getMarketplaceGlobalsByKey` is falsy: `mode: 'WEB'`